docs/one-click-integrations/jira-integration.md

```diff
@@ -58,6 +96,14 @@ The table below lists the data that the Jira integration collects from your Jira
     </tr>
 </table>
 
+## Which permissions does Pulse need from Jira? {: id="jira-permissions"}
+
+Pulse requests only the necessary permissions from Jira to collect issues data from your Jira instance and [keeps your information secure](https://security.codacy.com/). See below the detailed list of permissions.
+
+<!--TODO Add permissions table-->
```

Comment:
@afsmeira We currently have no information regarding the permissions that Pulse needs from Jira. Do we have any internal documentation that can help me to fill in this content?
If not, can you please place here the raw information about Jira permissions, so I can work on it? 🙏

Reply:
@afsmeira I added the permissions information. To make it simple, I'm including only the classic scopes. Let me know if it's ok like this, or if we need to add the granular scopes also.

Reply:
For me it's OK like this.